### ISSUE TITLE
don't use uncommon abbreviations in prop names

### ISSUE THREAD

**Issue by VladimirAlexiev** [Interoperable-data/ERA-Ontology-3.1.0#63]
(https://github.com/Interoperable-data/ERA-Ontology-3.1.0/issues/57 asks for labels to be pretty short. 
This is a "converse" issue).

Class and prop names should use only a limited number of well-known abbreviations.
To express the semantics more clearly, they should use words (while not being overly long). Eg:

Suggest to change these as follows:
- demonstrationINF -> demonstrationInfrastructureSubsysemCompliance
- demonstrationSRT-> demonstrationTunnelCompliance
- demonstrationENE -> demonstrationEnergySubsysemCompliance

There are also a number of props that haven't been deciphered from their relational source names (converting a name to conform to the naming convention, eg `dNvovtrp`, is not enough). These should be renamed to something more meaningful;
- D_NVOVTRP -> maximumDistanceForOverridingTrip
- D_NVPOTRP
- D_NVROLL
- Q_NVEMRRLS
- Q_NVDRIVER_ADHES
- Q_NVSBTSMPERM
- T_NVCONTACT
- T_NVOVTRP

Or let's take this example (closely related to https://github.com/Interoperable-data/ERA-Ontology-3.1.0/issues/62):
```ttl
era:etcsMVersion rdf:type owl:ObjectProperty ,
                 rdfs:range <http://www.w3.org/2004/02/skos/core#Concept> ;
                 era:inSkosConceptScheme <http://data.europa.eu/949/concepts/etcs-m-versions/ETCSMVersions> ;
                 era:XMLName "CPE_MVersion" ;
                 dcterms:requires "Only applicable when selected value for 1.1.1.3.2.1 (ETCS present)."@en ;
                 rdfs:comment "ETCS M_version according to SRS 7.5.1.9."@en ;
                 rdfs:seeAlso <https://www.era.europa.eu/domains/technical-specifications-interoperability/control-command-and-signalling-tsi_en#oe-content-paragraph-1632> ;
```
I'm trying to figure out what is "M"?
- https://data-interop.era.europa.eu/era-vocabulary/skos/era-skos-ETCSMVersions.ttl provides no explanation, since it merely collects the values 1.0, 1.1 etc
- "selected value for 1.1.1.3.2.1 (ETCS present)" is an applicability condition and doesn't explain what is M
- "SRS 7.5.1.9" is probably a reference to some specification that's probably part of some contract, but not public
- "CPE_MVersion" is part of an XML scheme but is also not public.

**Comment by Interoperable-data** [comment]
Dear @VladimirAlexiev , sometimes acronyms are understood by the railway experts. Many references to the documentations are all public available in the legislative text (as provided by the property `dcterms:source` or `rdfs:seeAlso `. We will try to make a balance regarding this suggestion, but we do not guarantee to fully implement this suggestion.

**Comment by VladimirAlexiev** [comment]
I agree with you! eg ETCS is very commonly used and it would be onerous to spell it in full. 
But please consider my other examples
